tests/integration/test_pqtools.ipynb

```diff
@@ -47,9 +48,22 @@
    "source": [
     "# Load temporary table\n",
     "\n",
-    "loaded_table = next(pq_reader.get_reader(\"uncategorized-assault\", tmp=True))\n",
+    "loaded_table = next(pq_reader.get_reader(temporary_table_name, tmp=True))\n",
     "loaded_table.info()"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Delete temporary table\n",
+    "\n",
+    "pq_writer.remove(temporary_table_name)\n",
+    "tmp_dir = data_dir.joinpath(\"tmp\")\n",
+    "tmp_dir.rmdir()"
```

Comment:
Used `Path.rmdir()` instead of `shutil.rmtree()` because an exception should be raised if another file was created in the `tmp` foler